lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -63,20 +59,14 @@ class Accessibility extends Gatherer {
     })()`;
 
     return driver
-        .evaluateAsync(expression)
-        .then(returnedValue => {
-          if (!returnedValue) {
-            return Accessibility._errorAccessibility('Unable to parse axe results');
-          }
-
-          if (returnedValue.error) {
```

Comment:
This appears to be from working around issues in the old `evaluateAsync` before @patrickhulce fixed up the error handling to properly reject. So, removed.